CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,12 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
-
-
+        public int Roll()
+        {
+            //Generate a random number 1 through 6 and returns it
+            var random = new Random();
+            var num = random.Next(1, 7);
+            return num;
+        }
     }
 }
```

Comment:
Commenting is good, sensible variable names, sticks to google c# style code

Reply:
Thank you :)